intern/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+from django.utils import timezone
+
+from core.models import User, Semester, Card
+from core.utils import get_semester
+from members.models import Member
+
+
+# Create your models here.
+class AccessLevel(models.Model):
+    name = models.CharField(max_length=20)
+    uio_name = models.CharField(max_length=50)
+    description = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+
+class InternGroup(models.Model):
+    name = models.CharField(max_length=50)
+    leader = models.ForeignKey(User)
+    description = models.CharField(max_length=300, null=True, blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Role(models.Model):
+    name = models.CharField(max_length=50)
+    description = models.CharField(max_length=300, null=True, blank=True)
+    groups = models.ManyToManyField(InternGroup, related_name='groups')
+    access_levels = models.ManyToManyField(AccessLevel)
+
+    def __str__(self):
+        return self.name
+
+
+
+class Intern(models.Model):
+    user = models.ForeignKey(User)
+    member = models.ForeignKey(Member, blank=True, null=True)
```

Comment:
Why do we have a reference to `Member`? We should be able to do this the other way around: `User` -> `Member`?